pkgs/development/interpreters/python/mk-python-derivation.nix

```
@@ -221,6 +222,7 @@ let
         else
           "setuptools";
 
+      relaxBuildDeps = finalAttrs.pythonRelaxBuildDeps or [ ] != [ ] || finalAttrs.pythonRemoveBuildDeps or [ ] != [ ];
```

This fixes the formatting of my previous suggestion.

```suggestion
      relaxBuildDeps =
        finalAttrs.pythonRelaxBuildDeps or [ ] != [ ] || finalAttrs.pythonRemoveBuildDeps or [ ] != [ ];

```